go/api/options/time_options.go

```diff
@@ -0,0 +1,7 @@
+package options
+
+import "github.com/valkey-io/valkey-glide/go/glide/api/config"
+
+type ClusterTimeOptions struct {
```

Comment:
Maybe we can define `RouteOption` and use it where applicable? We have similar thing in node client